src/Tizen.TV.UIControls.Forms/Handler/TVEditorHandler.cs

```diff
@@ -1,61 +1,50 @@
-﻿///*
-// * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
-// *
-// * Licensed under the Apache License, Version 2.0 (the License);
-// * you may not use this file except in compliance with the License.
-// * You may obtain a copy of the License at
-// *
-// * http://www.apache.org/licenses/LICENSE-2.0
-// *
-// * Unless required by applicable law or agreed to in writing, software
-// * distributed under the License is distributed on an AS IS BASIS,
-// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// * See the License for the specific language governing permissions and
-// * limitations under the License.
-// */
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-//using System;
-//using Microsoft.Maui.Controls;
-//using Microsoft.Maui.Handlers;
-//using TEntry = Tizen.UIExtensions.ElmSharp.Entry;
+using System;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+using TEditor = Tizen.UIExtensions.NUI.Editor;
 
-//namespace Tizen.TV.UIControls.Forms.Handler
-//{
-//    public class TVEditorHandler : EditorHandler
-//    {
-//        const string _doneKeyName = "Select";
-//        const string _cancelKeyName = "Cancel";
+namespace Tizen.TV.UIControls.Forms.Handler
+{
+    public class TVEditorHandler : EditorHandler
+    {
+        const string _doneKeyName = "Select";
```

Comment:
Does "Cancel" key no longer require?

Reply:
Good point! I double checked the Cancel process.
When it is implemented using ElmSharp, it was our responsibility to close keyboard by catching key event before cancel. However now on NUI, it is taken care of NUI, so we don't have to worry about this.